docker/terraform/Dockerfile

```diff
@@ -0,0 +1 @@
+FROM hashicorp/terraform:1.1.3 as terraform
```

Comment:
I was thinking of how to manage these referenced package versions.
so it's like using this as a configuration file ?

Reply:
per discussion offline, LGTM